app/portainer/components/sidebar/SidebarMenuItem/SidebarMenuItem.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import { Meta, Story } from '@storybook/react';
+
+import { SidebarMenuItem } from './SidebarMenuItem';
+
+const meta: Meta = {
+  title: 'Components/SidebarMenuItem',
+  component: SidebarMenuItem,
+};
+export default meta;
+
+interface StoryProps {
+  iconClass: string;
+  className: string;
+  itemName: string;
+  LinkName: string;
+}
+
+function Template({ iconClass, className, itemName, LinkName }: StoryProps) {
+  return (
+    <ul className="sidebar">
+      <div className="sidebar-list">
+        <SidebarMenuItem
+          path="example.path"
+          pathParams={{ endpointId: 1 }}
+          iconClass={iconClass}
+          className={className}
+          itemName={itemName}
+        >
+          {LinkName}
+        </SidebarMenuItem>
+      </div>
+    </ul>
+  );
+}
+
+export const Primary: Story<StoryProps> = Template.bind({});
+Primary.args = {
+  iconClass: 'fa-tachometer-alt fa-fw',
+  className: 'exampleItemClass',
+  itemName: 'ExampleItem',
+  LinkName: 'Example Link Name',
+};
+
+export function WithoutLinkName() {
```

Comment:
not sure I understand this story. do we have a use case of a sidebar item without a label?

Reply:
you're right, I don't think we have items anywhere with no name